assets/js/base/components/radio-control/style.scss

```diff
@@ -1,4 +1,119 @@
-@import "./mixin";
+.wc-block-components-radio-control__option {
+	@include reset-typography();
+	@include with-translucent-border( 0 0 1px );
+	display: block;
+	padding: $gap-small $gap-small 0 #{$gap-larger * 2};
+}
 
-@include radio-control-styles;
-@include radio-control-input-styles;
+.wc-block-components-radio-control__option-layout {
+	@include with-translucent-border( 0 0 1px );
+	display: table;
+	width: 100%;
+	padding-bottom: $gap-small;
+}
+
+.wc-block-components-radio-control__option .wc-block-components-radio-control__option-layout {
+	&::after {
+		display: none;
+	}
+}
+
+.wc-block-components-radio-control__label-group,
+.wc-block-components-radio-control__description-group {
+	display: table-row;
+
+	> span {
+		display: table-cell;
+	}
+
+	.wc-block-components-radio-control__secondary-label,
+	.wc-block-components-radio-control__secondary-description {
+		text-align: right;
+		min-width: 50%;
+	}
+}
+
+.wc-block-components-radio-control__label,
+.wc-block-components-radio-control__secondary-label {
+	line-height: 20px;
+	// Currently, max() CSS function calls need to be wrapped with unquote.
+	// See: https://github.com/sass/sass/issues/2378#issuecomment-367490840
+	line-height: unquote("max(1rem, 20px)");
+}
+
+.wc-block-components-radio-control__description,
+.wc-block-components-radio-control__secondary-description {
+	@include font-size( small );
+	line-height: 20px;
+}
+
+// Extra class for specificity.
+.wc-block-components-radio-control {
+	.wc-block-components-radio-control__input {
+		appearance: none;
+		background: transparent;
+		border: 2px solid currentColor;
+		border-radius: 50%;
+		display: inline-block;
+		height: 1.25rem;
+		left: $gap-large;
+		min-height: 20px;
+		min-width: 20px;
+		position: absolute;
+		top: $gap-small;
+		width: 1.25rem;
+
+		&:checked::before {
+			background: currentColor;
+			border-radius: 50%;
+			content: "";
+			display: block;
+			height: 0.625em;
+			left: 50%;
+			margin: 0;
+			min-height: 10px;
+			min-width: 10px;
+			position: absolute;
+			top: 50%;
+			transform: translate(-50%, -50%);
+			width: 0.625em;
+		}
+
+		.has-dark-controls & {
+			border-color: $controls-border-dark;
+
+			&:checked::before {
+				background: $input-text-dark;
+			}
+		}
+	}
+
+	@include breakpoint( ">782px" ) {
```

Comment:
Is there a specific reason why we don't use the `.is-large`?

Reply:
Hmmm, I thought the reason was that this component was built before we had the container-query classnames. But taking a second look but it seems like this media query makes the radio controls smaller. I think having bigger controls is intended to happen in handheld devices no matter the width of the block? So in this case it probably makes more sense using a CSS media query than container queries? :thinking:  I don't have a strong opinion, though.

I will merge this as-is, but I'm happy to change this in a follow-up if we agree on that.